lib/ClangImporter/IAMInference.cpp

```diff
@@ -165,6 +166,32 @@ static unsigned dropWordUniq(StringRef str, StringRef word, NameBuffer &out) {
   return numDropped;
 }
 
+static clang::Module *getSubmodule(const clang::NamedDecl *decl, clang::Sema &clangSema) {
```

Comment:
I bet there are a number of places in the Clang importer that should be making use of this, but that's a follow-up concern.